### PR TITLE
HYD-7298 Use EPEL as defacto dependency source

### PR DESCRIPTION
**Pull request by brianjmurrell**
We don't want the overhead of maintaining our own SRPMs, and building
IML from scratch.

Pull as many dependencies as possible from EPEL.

This involves converting over all necessary dependencies and then
auditing what's left and trying to convert any code needed so it is
compatible with EPEL versions.

The end goal is to have as little custom packaged source as possible.

Exceptions that need to be worked in separate tickets:
 - python-django
 - python-jsonschema

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>
Change-Id: I9a9512d1203d548333d73ee9399a7d7756ae6e30

### CODE REVIEW

File: chroma-dependencies/include/Makefile.lustre_build_meta.md5 [hunk 1]
```diff
@@ -1 +1 @@
-a8c027bf3d6266734d21e05354e15e02 -
+5732c91409d5fa5a39eb4540f8f4b25d -
```
Comment (jgrund) [inline chroma-dependencies/include/Makefile.lustre_build_meta.md5]: Got the ok to pull lustre and build on copr.

File: chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies [hunk 2]
```diff
@@ -1,5 +1 @@
 node-gyp-headers
```
Comment (jgrund) [inline chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies]: We should get rid of this as well.

File: chroma-dependencies/include/Makefile.lustre_build_meta.md5 (hunk 1 above)
Comment (brianjmurrell) [inline chroma-dependencies/include/Makefile.lustre_build_meta.md5]: Cool

File: chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies (hunk 2 above)
Comment (brianjmurrell) [inline chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies]: In this patch or another?

Comment (jgrund) [inline chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies]: Another is fine, don't want to hold this up.

Comment (brianjmurrell) [inline chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies]: So is this entirely extraneous as it is currently and can just be removed without anything else needing to be done or is there other things that need doing to be able to remove this?

Comment (jgrund) [inline chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies]: This was relevant when we were trying to build without an internet connection. 

This needs to be removed and the coorresponding dir in CE also needs removed (assume the later will just go when CE does).

File: chroma-bundles/include/find_deps.py [hunk 3]
```diff
@@ -92,9 +92,12 @@ def _find_deps(self, pkg, excludes):
 
     def disable_all_repos(self):
         for repo in self.yb.repos.findRepos("*"):
+            # these are hardcoded to the distro defaults and our own
+            # infrastructure right now
+            # these probably need to be externally specified
             if repo.id != "base" and repo.id != "core-0" and \
```
Comment (tanabarr) [inline chroma-bundles/include/find_deps.py]: Syntax would probably be nicer with repo.id not in [...] , definitely not worth repush

File: chroma-manager/chroma_core/lib/service_config.py [hunk 4]
```diff
@@ -268,8 +268,17 @@ def _setup_rabbitmq_service(self):
         #        blocking subprocess.communicate().
         #        we need to figure out why
         # FIXME: this should also be converted to use the common Shell utility class
-        self.try_shell(["service", "rabbitmq-server", "restart"],
-                       mystderr=None, mystdout=None)
+        #self.try_shell(["service", "rabbitmq-server", "restart"],
+        #               mystderr=None, mystdout=None)
+        # ServiceControlEL7 really needs a _restart() method
```
Comment (tanabarr) [inline chroma-manager/chroma_core/lib/service_config.py]: surprise that doesn't, will make a note to fix that

File: chroma-manager/tests/framework/utils/install_deps.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Copy the repo to the test node (don't want to have to clone every test run - impacts Gerrit)
```
Comment (tanabarr) [inline chroma-manager/tests/framework/utils/install_deps.sh]: probably want to remove reference to Gerrit given were not using any more

**Pull request closed without merging**